cmd/skopeo/proxy.go

```diff
@@ -204,6 +206,12 @@ func (h *proxyHandler) Initialize(args []any) (replyBuf, error) {
 	h.sysctx = sysctx
 	h.cache = blobinfocache.DefaultCache(sysctx)
 
+	policyContext, err := h.opts.global.getPolicyContext()
```

Comment:
For the record, caching it for the whole was already agreed upon as a possible improvement in https://github.com/containers/skopeo/pull/2029#discussion_r1252325538 .

The downside is that for a very long-running proxy, policy file changes would not affect the proxy.

Reply:
The proxy is not intended to run as a long-running separate daemon but be lifecycle bound to an invoking process (which typically should not also be long lived; rpm-ostree runs as a daemon but auto-exits on idle), so I think we're OK here.